schemas/Agreement.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Agreement.json",
+    "title": "Agreement",
+    "description": "Any type of agreement among economic agents.",
+    "type": "object",
+    "properties": {
+        "name": {
+            "description": "An informal or formal textual identifier for the item. Does not imply uniqueness.",
```

Comment:
In most cases, you are repeating the schema in the name description. In this case, just "for the item", not "for the Agreement".